core-services/prow/02_config/_plugins.yaml

```diff
@@ -4400,6 +4422,31 @@ plugins:
   - verify-owners
   - wip
   - yuks
+  openshift-priv/cluster-api-provider-kubevirt:
```

Comment:
I think you should rename `openshift-priv` to `openshift` and rerun `make prow-config`